x-pack/test/api_integration/apis/metrics_ui/inventory_threshold_alert.ts

```diff
@@ -77,51 +80,248 @@ export default function ({ getService }: FtrProviderContext) {
     startTime: DATES['8.0.0'].hosts_only.max,
   };
 
+  // TODO: Need a test for: host rx, pods rx, logRate
```

Comment:
@simianhacker Do we need to test other default metrics as well (for example outbound traffic, load etc)? Do they have something special that needs to be tested out?

Reply:
There are really 3 classes of aggregations in this system:
1. Log Rate which takes the document count and divides it by the interval
2. Network Traffic with/without interfaces which creates 2 buckets of data and calculates the derivative normalized by the interval to give us a per second rate. This also requires us to fetch double the amount of data to fill both buckets.
3. Basic metric aggregations which only needs 1 bucket of data and runs basic aggregations like `avg`, `max`, `min`, and `sum`. The CPU usage aggregation is, technically, a little more complex because it uses a `bucket_script` to do some basic math between 3 metrics but it's bucketing requirements are similar to a basic metric aggregation like `avg`.

Reply:
Oh... I need to add a test for "rates without interfaces".

Reply:
Thanks a lot for the explanation!